scripts/sender.py

```diff
@@ -0,0 +1,119 @@
+"""
+This module contains a simple class for sending Accounting Records.
+
+Accounting Records are sent to a REST endpoint.
+If run as a python script, this module will
+send messages to the REST endpoint defined in main().
+"""
+
+
+import httplib
+import json
+import logging
+import ssl
+import socket
+import sys
+import time
+
+from dirq.QueueSimple import QueueSimple
+
+log = logging.getLogger(__name__)
+
+
+class Sender(object):
+    """A simple class for sending Accounting Records to a REST endpoint."""
+
+    def __init__(self, dest, qpath, cert, key):
+        """Initialize a Sender."""
+        self._cert = cert
+        self._key = key
+        self._outq = QueueSimple(qpath)
+        self._dest = dest
+
+    def send_all(self):
+        """Send all the messages in the outgoing queue via REST."""
+        log.info('Found %s messages.', self._outq.count())
+        for msgid in self._outq:
+            if not self._outq.lock(msgid):
+                log.warn('Message was locked. %s will not be sent.', msgid)
+                continue
+
+            text = self._outq.get(msgid)
+
+            url = '/api/v1/cloud/record'
```

Comment:
the variable should probably be renamed `path`.

the version number can be set in the config, but the path (excluding version) will always be the same.